internal/controller/bucket_controller.go

```diff
@@ -147,6 +146,14 @@ func (r BucketReconciler) reconcileNormal(ctx context.Context, objectStorageServ
 func (r BucketReconciler) reconcileDelete(ctx context.Context, objectStorageService objectstorage.ObjectStorageService, accessRoleService objectstorage.AccessRoleService, bucket *v1alpha1.Bucket) error {
 	logger := log.FromContext(ctx)
 
+	// TODO(quentin) Remove this, this is needed now for the glippy tests.
+	logger.Info("Configuring bucket settings")
```

Comment:
This will go away, I just need to have the lock tested on a bucket cr stuck in deleting state on glippy